lib/core/cardano-wallet-core.cabal

```diff
@@ -203,6 +203,7 @@ test-suite unit
       Cardano.Wallet.ApiSpec
       Cardano.Wallet.DB.Arbitrary
       Cardano.Wallet.DB.MVarSpec
+      Cardano.Wallet.DB.PoolProperties
```

Comment:
--> Cardano.Pool.DB.Properties, let keep the same namespaces as the sources :+1:

Reply:
moved